about.html

```diff
@@ -1,57 +1,87 @@
 <!DOCTYPE html>
-<html> 
-<head>
-    <title>JumboCode: About</title>
-    <meta charset="utf-8">
-    <meta name="description" content="About Us Page of the JumboCode website">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-</head>
-
-<body>
-
-    <div>
-        <h2>Who We Are</h2>
-    </div>
 
-    <div>
-        <h2>How We Operate</h2>
-    </div>
-
-    <div>
-        <h2>Executive Board</h2>
-        <img src="image location" />
-        <h3>Yuki Zaninovich</h3>
-        <p>President</p>
+<html>
+<head>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Arimo:400,700" rel="stylesheet">
 
-        <img src="image location" />
-        <h3>Hayley Cohen</h3>
-        <p>Client Manger</p>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-color/2.1.2/jquery.color.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/2.6.3/svg.js"></script>
 
-        <img src="image location" />
-        <h3>Andrew Hoiberg</h3>
-        <p>Project Manger</p>
+	<link rel="stylesheet" href="styles.css">
+	<script src="scripts.js"></script>
 
-        <img src="image location" />
-        <h3>Ben Cresitello-Dittmar</h3>
-        <p>Project Manger</p>
+</head>
+<body>
+	<div class="right-navigation">
+		<ul>
+			<li><a href="">Contact</a></li>
```

Comment:
Please add the links to the other pages, it should just be `<a href="contact.html">` etc. since all the files are in the main directory